bin/setup

```diff
@@ -30,47 +22,56 @@ end.parse!
 # path to your application root.
 APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
 
-Dir.chdir APP_ROOT do
-  puts "== Installing dependencies =="
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+chdir APP_ROOT do
```

Comment:
This is kind of a "half-:trollface:"/half serious.

Kinda curious why the Rails team went with the `Fileutils::chdir` instead of the `Dir.chdir`.  [Under the hood](http://ruby-doc.org/stdlib-1.9.3/libdoc/fileutils/rdoc/FileUtils.html#method-c-chdir), it is using `Dir.chdir` anyway.  It is not like we are passing any fancy options to it, so essentially we are "bloating" (I use that term loosely since this is a script we are talking about here...) the method list to use one method, and it doesn't even save us characters or even become more or less readable.

Anyway, seems odd, so something I thought I would point out.